cmd/dep/vndr_importer.go

```diff
@@ -103,10 +118,27 @@ func (v *vndrImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, er
 				ProjectRoot: gps.ProjectRoot(pkg.importPath),
 				Source:      pkg.repository,
 			},
+			Constraint: gps.Any(),
 		}
-		pc.Constraint, err = v.sm.InferConstraint(pkg.revision, pc.Ident)
+
+		isVersion, version, err := isVersion(pc.Ident, pkg.reference, v.sm)
```

Comment:
A comment that a reference could be a version or revision. So, we need to first check if the reference is a version would be helpful.
I got confused for the need of this check 😅 then looked up an example vendor.conf file.

Reply:
Yeah, the naming is confusing. I didn't even realize that a version could be stored in that field, which is why I renamed it to reference. I'll comment it up. 😀